### ISSUE TITLE
Provide better message when pytest receives invalid filter with `-W`

### ISSUE THREAD

**Issue by nicoddemus** [pytest-dev/pytest#8096]
Your `-ArgumentList $myparams -Wait -PassThru` causes `-Wait` to be passed to pytest - that in turn passes it to Python's `warnings` module as an `ait` warning filter, which isn't valid.

I suppose pytest could show a better error message in this case though.

_Originally posted by @The-Compiler in https://github.com/pytest-dev/pytest/issues/8095#issuecomment-737862151_

**Comment by AirbusA321NX** [comment]
hey, i looked into the issue and was able to reproduce it on my side

so basically whats happening is -Wait is getting parsed by pytest as -W (short option for --pythonwarnings) and the rest becomes ait, which then leads to the confusing error invalid action: 'ait'...

i think we can make this a lot clearer for users by improving the error message when this specific kind of parsing mistake happens

I think this the solution that is required...

in src/_pytest/config/__init__.py, inside parse_warning_filter, when _getaction throws OptionError, we can keep the existing behavior but add a small hint when this looks like a shell argument issue (like -Wait being split)

something like

try:
    action: warnings._ActionKind = warnings._getaction(action_)
except warnings._OptionError as e:
    msg = str(e)
    if action_ == "ait":
        msg += ". note that '-W' is a short option for '--pythonwarnings', so it might be eating part of a shell argument like '-Wait'"
    raise UsageError(error_template.format(error=msg)) from None


in short what i mean is instead of just showing a cryptic invalid action: ait error, we give a small hint that this can happen when shell args get misinterpreted, especially things like powershell -Wait

if its a good approach tell me , i will start working on it. 
thx
